topics/reverse-engineering/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: reverse

```

Comment:
This alias seems a little too broad. Thoughts on removing it?
```suggestion
```

Reply:
Suggestion taken into account. In fact this a little to large. I removed it